models/clm/readme.rst

```diff
@@ -253,16 +375,55 @@ The following is perfectly legal (for CLM4, at least):
                    'TLAI',       'QTY_LEAF_AREA_INDEX', 'NA', 'NA', 'vector'  , 'NO_COPY_BACK'
       /
 
-however, only LAIP_VALUE will be used to calculate the LAI when an observation of LAI is encountered. All the other LAI
-variables in the DART state will be modified by the assimilation based on the relationship of LAIP_VALUE and the
-observation. Those coming from the restart file and marked 'UPDATE' **will** be updated in the CLM restart file.
+however, only **LAIP_VALUE** will be used to calculate the LAI when an 
+observation of LAI is encountered. **All** (the other LAI) variables in 
+the DART state will be modified by the assimilation based on the 
+relationship of LAIP_VALUE and the observation. 
+
+Inflation
+---------
+
+Inflation has been shown to be quite useful in our experience of
+DA with CLM and DART. The model is strongly influenced by the
+atmospheric forcing and will cause the CLM ensemble to
+relax to a state consistent with the forcing when the assimilation
+stops. [5]_ Depending on the forecast length between assimilations, and 
+sometimes just to restore the variance lost during an assimilation, 
+inflation should be used.
+
+The 'NO_COPY_BACK' designation has some side effects when it 
+comes to state-space inflation (inf_flavor 2,4 or 5 - 
+'VARYING_SS_INFLATION','RELAXATION_TO_PRIOR_SPREAD', 
+or 'ENHANCED_SS_INFLATION' - respectively).  State-space inflation
+requires an inflation value for everything in the DART state. 
+If the variable has been designated as 'NO_COPY_BACK'
+the DART write routine (when called from ``filter``) simply 
+skips the variable and nothing is written.
+This is a problem for inflation files that need to adapt.
+
+The solution is to run 
+:doc:`../../assimilation_code/programs/fill_inflation_restart/fill_inflation_restart`
+to create an initial inflation file with inflation values of 1.0 (i.e.
+no inflation). ``fill_inflation_restart`` has been specially designed
+to output inflation values for every variable in the DART state. 
+The idea is to copy the *input* inflation file to the *output* inflation
+file name *before each assimilation cycle*. No new values will be written
+for the variables designated 'NO_COPY_BACK', the original values will persist. 
+
```

Comment:
This is a different 'fix' than turning off the adaptive inflation for the first assimilation time step, then turning it on for the remainder of the assimilation window?    Not exactly clear how fill_inflation_restart is implemented as it doesn't appear to be a namelist option.